frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -59,17 +65,18 @@ export const ClusterServiceVersionRow = withFallback<ClusterServiceVersionRowPro
         <ClusterServiceVersionLogo icon={_.get(obj, 'spec.icon', [])[0]} displayName={obj.spec.displayName} version={obj.spec.version} provider={obj.spec.provider} />
       </Link>
     </div>
-    <div className="col-xs-3">
+    <div className="col-xs-2">
       <ResourceLink kind="Namespace" title={obj.metadata.namespace} name={obj.metadata.namespace} />
     </div>
     <div className="col-xs-2">
       <ResourceLink kind="Deployment" name={obj.spec.install.spec.deployments[0].name} namespace={obj.metadata.namespace} title={obj.spec.install.spec.deployments[0].name} />
     </div>
     <div className="col-xs-2">{obj.metadata.deletionTimestamp ? 'Disabling' : installStatus}</div>
-    <div className="col-xs-2">
-      <div style={{display: 'flex', justifyContent: 'flex-end', alignItems: 'center'}}>
-        <Link to={`${route}/instances`} title="View instances">View instances</Link>
-      </div>
+    <div className="col-xs-3">
+      { _.take(obj.spec.customresourcedefinitions.owned, 4).map(desc => <div key={desc.name}>
+        <Link to={`${route}/instances`} title={desc.name}>{desc.displayName}</Link>
+      </div>)}
+      { obj.spec.customresourcedefinitions.owned.length > 5 && <Link to={`${route}/instances`} title={`View ${obj.spec.customresourcedefinitions.owned.length - 4} more...`}>{`View ${obj.spec.customresourcedefinitions.owned.length - 4} more...`}</Link>}
```

Comment:
It looks like if there are exactly 5 we'll drop one without displaying the message